src/terra-dev-site/doc/application/reference.1/components/ApplicationNavigation.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import PropsTable from 'terra-application/lib/application-navigation/ApplicationNavigation?dev-site-props-table';
+import ApplicationNavigationExample from 'terra-application/lib/terra-dev-site/doc/application/reference.1/components/example/ApplicationNavigationExample?dev-site-example';
+
+# ApplicationNavigation
+
+The ApplicationNavigation component provides a responsive layout and a variety of controls for application navigation.
```

Comment:
😄 can we go into detail how it's responsive and what types of controls are provided? I know the props table will give options, but it'd be easier to understand what you get from this if you can see a face-up list

Reply:
I'll go ahead and copy forward a lot of the docs from the original package. It'll give helpful context to a lot of these concepts.